TSPL.docc/LanguageGuide/OpaqueTypes.md

```diff
@@ -921,6 +921,63 @@ which means that the type of `twelve` is also inferred to be `Int`.
   }
 -->
 
+
+## Opaque Parameter Types
+
+In addition to writing `some` to return an opaque type,
+you can also write `some` in the type for a parameter
+to a function, subscript, or initializer.
+However, when you write `some` in a parameter type
+that's just a shorter syntax for generics, not an opaque type.
+For example,
+both of the functions below are equivalent:
+
+```swift
+func drawTwiceGeneric<SomeShape: Shape>(_ shape: SomeShape) -> String {
+    let drawn = shape.draw()
+    return drawn + "\n" + drawn
+}
+
+func drawTwiceSome(_ shape: some Shape) -> String {
+    let drawn = shape.draw()
+    return drawn + "\n" + drawn
+}
+```
+
+The `drawTwiceGeneric(_:)` function
+uses a generic type parameter named `SomeShape`,
+and explicitly includes the protocol conformance requirement.
```

Comment:
s/includes/states/ ?

Also do we call them "requirements" or "constraints" here? You also use "constraint" below